rts/Sim/Misc/ModInfo.h

```diff
@@ -195,6 +195,9 @@ class CModInfo
 	/// Point at which a region is considered bad for raw path tracing.
 	float pfRawMoveSpeedThreshold;
 
+	/// Increases Hcost in QTPFS pathing. 0.0 = no increase, 1.0 = increase to the mean move cost, 2.0 increase to max move cost.
```

Comment:
nobody (and especially not game devs) knows what `Hcost` is. ideally this would give the underlying principle (0 means the unit chooses the bazzest path, 2 means the unit chooses the quuxest path) and/or an example (if a unit can choose between path of length X and speed mod Y or path of length Z and speed mod Q then with 2 it will choose...)

additionally there could be something about intermediate values, what if i put 0.3 there? (as far as i can tell it's a linear interpolation, but i had to find it in the code so saying it explicitly would be nice)

Reply:
I've added a more detailed explaination. This isn't a simple concept to explain, but hopefully that goes some way to helping.